lib/background-worker/background-worker-props.ts

```diff
@@ -1,5 +1,53 @@
 import { IoK8SApiCoreV1PodSpecRestartPolicy } from "../../imports/k8s";
 import { ContainerProps, WorkloadProps } from "../common";
+import { RedisConnectionDetails } from "../redis/redis-util";
+
+export interface RedisListScalerProps {
+  /** Name of the Redis List that you want to monitor. */
+  readonly listName: string;
+
+  /** Average target value to trigger scaling actions. */
+  readonly listLength: number;
+
+  /** Redis connection details. */
+  readonly redisConnectionDetails: RedisConnectionDetails;
+}
+
+export interface BackgroundWorkerAutoscalingProps {
+  /**
+   * Whether to pause autoscaling.
+   * When set to true, `minReplicas` will be used as the desired replica count.
+   * @see https://keda.sh/docs/2.9/concepts/scaling-deployments/#pause-autoscaling
+   */
+  readonly paused?: boolean;
+
+  /**
+   * Minimum number of replicas.
+   * @default 0
+   */
+  readonly minReplicas?: number;
+
+  /**
+   * Maximum number of replicas.
+   * @default 100
+   */
+  readonly maxReplicas: number;
+
+  /**
+   * Interval to check each scaler on.
+   * @default 30
```

Comment:
I wonder if this should be lower? But I guess that comes with an additional cost

Reply:
That's KEDA's default when there's no value set, and I'm not setting it.
I'd leave it - we can fine tune it on a need basis, or, if we find that we often need to tune it, we can revisit here.